app/models/membership.rb

```diff
@@ -80,5 +85,6 @@ def unarchive!
   def accept!
     newbie_post = posts.find { |post| post.post_type.id == DEFAULT_POST_ID }
     newbie_post.destroy
+    post_types << PostType.find(NEW_MEMBER_ID)
```

Comment:
I think if we `accept!` multiple times, then the memberships has multiple new member post.

Reply:
If you think I can add a guard clause, but this method can be called only once during 'normal usage'. I think there is a constraint in the database too, but I have to check it.

Reply:
@emanreyalp I would leave it like this. As I've mentioned in the comment above, this method should be called once as the button is shown while the membership is not accepted. For multiple calls it will fail earlier, because `destroy` will be called on `nil` and I think it is the required behavior, because we will get informed through Rollbar.

Reply:
You are right. Not this is the right place where we have to check the duplication.
I checked the structure and there isn't any unique constraint on `poszt` table with `grp_member_id`  and `pttip_id`.
If you didn't create that in this PR (and the model validation too) then please create a card for this task.